scripts/scil_snr_in_roi.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Script to compute signal to noise ratio (SNR) in a region of interest (ROI)
+of a DWI volume.
+
+It will compute the SNR for all DWI volumes of the input image seperately.
+The output will contain the SNR.
+The mean of the signal is computed inside the mask.
+The standard deviation of the noise is estimated inside the noise_mask
+or inside the same mask if a noise_map is provided.
+If it's not supplied, it will be estimated using the data outside the brain,
+computed with Dipy medotsu
+
+If verbose is True, the SNR for every DWI volume will be outputed.
+
+This works best in a well-defined ROI such as the corpus callosum.
+It is heavily dependent on the ROI and its quality.
```

Comment:
(nouveau paragraphe)
We highly recommend using a noise_map if you can acquire one. See refs [1, 2] that describe the noise map acquisition. 
[1] St-Jean, S., Coupé, P., & Descoteaux, M. (2016). Non Local Spatial and Angular Matching : Enabling higher spatial resolution diffusion MRI datasets through adaptive denoising. Medical Image Analysis, 32, 115–130. https://doi.org/10.1016/j.media.2016.02.010
[2]  Reymbaut, A., Caron, A. V., Gilbert, G., Szczepankiewicz, F., Nilsson, M., Warfield, S. K., Descoteaux, M., & Scherrer, B. (2021). Magic DIAMOND: Multi-fascicle diffusion compartment imaging with tensor distribution modeling and tensor-valued diffusion encoding. Medical Image Analysis, 70. https://doi.org/10.1016/j.media.2021.101988